deploy.sh

```diff
@@ -15,6 +15,7 @@ if [ -z ${D4M_HOME+x} ]; then
   echo "Not installing Graphulo JAR in D4M installation because D4M_HOME is not set";
 else
   cp $(echo target/graphulo-*.jar | tr ' ' '\n' | grep -v alldeps) "$D4M_HOME/lib"
+  cp -f target/graphulo-*-libext.zip $D4M_HOME
```

Comment:
Why put the zip file in `D4M_HOME`?

Reply:
The ant build unzips it and puts it in the final d4m_api zip.

Reply:
Okay, that means there is overlap between this deploy script and the ant script. What does the ant script do that the deploy script does not do?

Reply:
I see what you're saying, I didn't see the line under where it unzips it into D4M_HOME.

The ant script runs in the d4m folder and creates a clean d4m_api zipped folder containing only what you need to run D4M, with the graphulo jar and libext folder if it's there. I don't run the deploy.sh script because I don't have access to ACCUMULO_HOME or D4M_HOME. Other than that, I'm not sure it does much else, but then again I don't know a ton about ant.

Reply:
Great, it looks like the deploy script can replace the ant script.  I would remove this line.

For users who want to use the ant script for some reason (e.g., maybe they are on Windows and the shell script doesn't work), they can manually copy the libext zip to their D4M directory. No need for them to know about D4M_HOME.

Reply:
Is there a way deploy.sh can check that the user has permission to copy the jars to the folders specified? Right now it just checks whether D4M_HOME and ACCUMULO_HOME are set, and then tries to copy to those locations, which most likely the user won't have access to on a shared system like E1.

I'm asking Chuck about the ant script.

Reply:
I've removed that line and made the changes that handle permissions issues.